samples/tab-mail/ts/README.md

```diff
@@ -0,0 +1,93 @@
+---
+page_type: sample
+description: Sample tab-mail app.
```

Comment:
This sample app shows how to compose mail in Outlook using personal tab app.

Reply:
fixed